File: LorePdks.COMMON/DTO/Services/DataTable/DataTableResponseDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LorePdks.COMMON.DTO.Services.DataTable
{
    public class DataTableResponseDTO<T>
    {
        public List<T> data { get; set; }
        public long draw { get; set; }
        public long recordsFiltered { get; set; }
        public long recordsTotal { get; set; }
    }
    public class DataTableResponse
    {
        public List<dynamic> data { get; set; }
        public long recordsTotal { get; set; }
    }
}

File: LorePdks.WEB/src/app/core/models/DataTableResponseDto.ts
/**
 * DataTable yanıt bilgilerini temsil eden model
 */
export class DataTableResponseDto<T> {
  data: T[];
  draw: number;
  recordsFiltered: number;
  recordsTotal: number;
}

/**
 * DataTable genel yanıt bilgilerini temsil eden model
 */
export class DataTableResponse {
  data: any[];
  recordsTotal: number;
}